timeline-chart/src/layer/time-graph-chart-cursors.ts

```diff
@@ -287,4 +301,28 @@ export class TimeGraphChartCursors extends TimeGraphChartLayer {
             delete this.secondCursor;
         }
     }
+
+    destroy() : void {
+        super.destroy();
```

Comment:
asking, should this be at the end of the "destroy"?

Reply:
@MatthewKhouzam hmm, interesting point. I think it's better to clean-up this class first before call destroy on parent. I'll change it here and other relevant place in this PR.